src/components/Checkbox/index.tsx

```diff
@@ -56,14 +56,10 @@ const StyledReakitCheckbox = styled(ReakitCheckbox, {
   shouldForwardProp: prop => !['size'].includes(prop),
 })`
   position: absolute;
-  cursor: pointer;
-  width: 1px;
-  height: 1px;
-  padding: 0;
-  margin: -1px;
-  overflow: hidden;
-  clip: rect(0, 0, 0, 0);
   white-space: nowrap;
+  height: ${({ size }) => size}px;
+  width: ${({ size }) => size}px;
+  opacity: 0;
```

Comment:
If I remember correctly, opacity 0 on safari and some other browser is considered non present element, so you won't be able to click on the checkbox

Reply:
@chambo-e it should be fine it is the label who's clickable, the input is inside the label